app/grandchallenge/annotations/management/commands/copyannotations.py

```diff
@@ -59,12 +81,26 @@ def handle(self, *args, **options):
                 obj.grader_id = user_to.id
                 obj.pk = None
                 obj.save()
+                if options["add_permissions"]:
+                    for permission_type in PERMISSION_TYPES:
+                        assign_perm(
+                            f"annotations.{permission_type}_{model.__name__.lower()}",
```

Comment:
Why use `model.__name__` here and `model.__class__.__name__` below? Should we be using `instance._meta.app_label`, and `instance._meta.model_name` instead?

Reply:
Done in 3ce431a984d771bda25d6afd42fbf573f887e140